src/test/java/com/gfl/client/model/ProxyCredentialsTest.java

```diff
@@ -0,0 +1,93 @@
+package com.gfl.client.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyCredentialsTest {
+    private ProxyCredentials proxyCredentials;
+
+    @BeforeEach
+    void setUp(){
+        proxyCredentials = new ProxyCredentials("username1", "password1");
+    }
+
+    @Test
+    @DisplayName("Test empty constructor with null password and username")
+    void testEmptyConstructor(){
+        ProxyCredentials emptyProxyCredentials = new ProxyCredentials();
+
+        assertNull(emptyProxyCredentials.getUsername());
+        assertNull(emptyProxyCredentials.getPassword());
+    }
+
+    @Test
```

Comment:
these test cases can be changed to `@ParameterizedTest`.
you can use the `ProxyCredentialsArgumentsProvider`